src/Popover/Popover.js

```diff
@@ -168,9 +185,15 @@ export default class Popover extends Component<Props, State> {
             this.popoverContent = node;
           },
           subtitle,
+          tabIndex: 0,
           title
         };
 
+        popoverContentProps = {
+          ...popoverContentProps,
+          ...(getContentProps && getContentProps(popoverContentProps))
```

Comment:
because of the function call, `getContentProps(popoverContentProps))` needs the contentProps as input. @brentertz has a more streamlined composer coming with select.

Reply:
Oh, duh. I obviously missed that. 🤦‍♂️

Reply:
@brentertz do you want to drop that util as a chore on master so I can just use it?

Reply:
Don't sweat the new pattern.  This code is fine for now.